pkg/cmd/server/kubernetes/node/node_config.go

```diff
@@ -325,24 +330,17 @@ func BuildKubernetesNodeConfig(options configapi.NodeConfig, enableProxy, enable
 			dnsConfig.Nameservers = options.DNSNameservers
 		}
 
-		services, serviceStore := dns.NewCachedServiceAccessorAndStore()
-		endpoints, endpointsStore := dns.NewCachedEndpointsAccessorAndStore()
-		if !enableProxy {
```

Comment:
what was the intent of doing this differently if the proxy wasn't enabled? why is that no longer necessary?